src/test/java/apoc/export/cypher/ExportCypherTest.java

```diff
@@ -67,7 +70,50 @@
             "begin%n" +
             "DROP CONSTRAINT ON (node:`UNIQUE IMPORT LABEL`) ASSERT node.`UNIQUE IMPORT ID` IS UNIQUE;%n" +
             "commit");
-private static final Map<String, Object> exportConfig = Collections.singletonMap("separateFiles", true);
+
+    private static final String EXPECTED_CYPHER_SHELL = String.format(":begin%n" +
```

Comment:
don't copy & paste it, combine it from the same fragements that are then just joined with the proper begin/commit/await pieces.